cypress/e2e/shared/checks.test.ts

```diff
@@ -10,14 +10,17 @@ describe('Checks', () => {
     cy.flush()
 
     cy.signin().then(() => {
-      cy.writeData([`${measurement} ${field}=0`, `${measurement} ${field}=1`])
-
       // visit the alerting index
-      cy.get('@org').then(({id}: Organization) =>
+      cy.get('@org').then(({id: orgID, name: orgName}: Organization) => {
+        cy.writeData(
+          [`${measurement} ${field}=0`, `${measurement} ${field}=1`],
+          orgName,
```

Comment:
why not have writeData run the `cy.get('@org')` so that we can simplify the interface?

Reply:
ok